pybind11/setup_helpers.py

```diff
@@ -306,24 +324,42 @@ class ParallelCompile(object):
     This takes several arguments that allow you to customize the compile
     function created:
 
-    envvar: Set an environment variable to control the compilation threads, like NPY_NUM_BUILD_JOBS
-    default: 0 will automatically multithread, or 1 will only multithread if the envvar is set.
-    max: The limit for automatic multithreading if non-zero
+    envvar:
+        Set an environment variable to control the compilation threads, like
+        NPY_NUM_BUILD_JOBS
+    default:
+        0 will automatically multithread, or 1 will only multithread if the
+        envvar is set.
+    max:
+        The limit for automatic multithreading if non-zero
+    needs_recompile:
+        A function of (obj, src) that returns True when recompile is needed.  No
+        effect in isolated mode; use ccache instead, see
```

Comment:
If you want another nitpicky remark on capitals: "CCache"?

Reply:
Actually, apparently it's "Ccache" or "ccache", according to their own website: https://ccache.dev/ So maybe the other ones should be changed?

(Or just ignore this; I'm sure there's better ways to spend your time.)

Reply:
Dissalowing CCache, and fixed.